docs/en_US/reference/experiment_config.rst

```diff
@@ -370,41 +381,42 @@ See algorithm's document for supported value.
 TrainingServiceConfig
 ^^^^^^^^^^^^^^^^^^^^^
 
-One of following:
+One of the following:
 
 - `LocalConfig`_
 - `RemoteConfig`_
-- `OpenpaiConfig <openpai-class>`_
+- :ref:`OpenpaiConfig <openpai-class>`
 - `AmlConfig`_
+- `HybridConfig`_
 
-For other training services, we suggest to use `v1 config schema <../Tutorial/ExperimentConfig.rst>`_ for now.
+For `Kubeflow <../TrainingService/KubeflowMode.rst>`_, `FrameworkController <../TrainingService/FrameworkControllerMode.rst>`_, and `AdaptDL <../TrainingService/AdaptDLMode.rst>`_ training platforms, NNI suggests to use `v1 config schema <../Tutorial/ExperimentConfig.rst>`_ for now.
 
 
 LocalConfig
-^^^^^^^^^^^
+-----------
 
 Detailed `here <../TrainingService/LocalMode.rst>`__.
 
 platform
---------
+""""""""
 
 Constant string ``"local"``.
 
 
 useActiveGpu
-------------
+""""""""""""
 
 Specify whether NNI should submit trials to GPUs occupied by other tasks.
 
 type: ``Optional[bool]``
 
 Must be set when `trialGpuNumber` greater than zero.
 
-If your are using desktop system with GUI, set this to ``True``.
+If you are using the desktop system with GUI, set this to ``True``.
```

Comment:
Please add these detail info:

Following processes can make GPU "active":
  - non-NNI CUDA programs
  - graphical desktop
  - trials submitted by other NNI instances, if you have more than one NNI experiments running at same time
  - other users' CUDA programs, if you are using a shared server

Therefore, if you are using graphical OS like Windows 10 or Ubuntu desktop and `useActiveGpu` is set to `True`, the GUI will prevent NNI from launching any trial.
If you creates multiple NNI experiments and `useActiveGpu` is set to `False`, they will submit multiple trials to one GPU.

Reply:
... and `useActiveGpu` is set to **`False`**, the GUI will prevent NNI from launching any trial?

Reply:
Yay, they were flipped.

Reply:
If I create multiple NNI experiments and `useActiveGpu` is set to `True`, only use one GPU?

Reply:
1. Assume there is only one GPU on the machine, then each experiment will submit one trial to that GPU. The GPU will have 2 trials run concurrently.
2. Set `trialGpuNumber`, that's all. Generally you should not need fine-grained control like `useActiveGpu` on "managed" platforms.
3. Yes, `gpuIndices: 0,1,2` is string, `gpuIndices: 0,1` is string, but `gpuIndices: 0` is int.